src/ruby/detectors/code-injection/code-injection.rb

```diff
@@ -0,0 +1,20 @@
+=begin
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+=end
+
+# {fact rule=code-injection@v1.0 defects=1}
+def code_injection_noncompliant()
+  code = params[:code]
+  # Noncompliant: 'code' is not sanitized.
```

Comment:
Which one is the user input? code? args?

Reply:
the args is the user input. update the description.

Reply:
Non compliant example should send `args` instead of `code`?